tests/kafkatest/services/templates/verifiable_producer.properties

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I discussed this offline with @guozhang - to avoid having empty dummy files floating around, it may be better to delete this file and pass in empty string to in the `SecurityConfig` constructor in `verifiable_producer` (see comment below)